src/styles.scss

```diff
@@ -1,32 +1,4 @@
-/* You can add global styles to this file, and also import other style files */
-@font-face {
-    font-family: 'plentyicons';
-    src: url('/assets/fonts/plentyicons.eot?2x5xl1');
-    src: url('/assets/fonts/plentyicons.eot?2x5xl1#iefix') format('embedded-opentype'),
-        url('/assets/fonts/plentyicons.ttf?2x5xl1') format('truetype'),
-        url('/assets/fonts/plentyicons.woff?2x5xl1') format('woff'),
-        url('/assets/fonts/plentyicons.svg?2x5xl1#plentyicons') format('svg');
-    font-weight: normal;
-    font-style: normal;
-}
-
-[class^='icon-'],
-[class*=' icon-'] {
-    /* use !important to prevent issues with browser extensions that change fonts */
-    font-family: 'plentyicons' !important;
-    speak: none;
-    font-style: normal;
-    font-weight: normal;
-    font-variant: normal;
-    text-transform: none;
-    line-height: 1;
-
-    /* Better Font Rendering =========== */
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-}
-
```

Comment:
das steckt dann in der styles.css aus dem terra-icons repo?

Reply:
yes